admin_guide/topics/about-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='about-cluster-auto-scaler-{context}']
+= About the {product-title} auto-scaler
+
+The auto-scaler in {product-title} repeatedly checks to see how many
+pods are pending node allocation. If pods are pending allocation and
+the auto-scaler has not met its maximum capacity, then new nodes are
+continuously provisioned to accommodate the current demand. When
+demand drops and fewer nodes are required, the auto-scaler removes
+unused nodes. After you install the auto-scaler, its behavior is automatic. You
+only need to add the desired number of replicas to the deployment.
+
+In {product-title} version {product-version}, you can deploy the
+auto-scaler only on Amazon Web Services (AWS). The auto-scaler uses
+some standard AWS objects to manage your cluster size, including Auto
+Scaling groups (ASGs) and Launch Configurations (LCs).
+
+The auto-scaler uses the following assets:
+
+Auto Scaling groups:: An ASG is a logical
```

Comment:
@kalexand-rh Did you check the AWS docs for this abbreviation? I took a quick look and didn't see it. It looks a bit odd that _groups_ is small G when spelled out; but becomes uppercase in the abbreviation. Doesn't affect understanding. Just made me question if the name is _Auto Scaling Group_ (which it is not).

Reply:
Good point. I can't find ASG in the docs, but they do use Auto Scaling group :https://docs.aws.amazon.com/autoscaling/ec2/userguide/create-asg-launch-template.html